examples/apps/Dockerfile.centrifugo

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.16
+
+RUN git clone https://github.com/centrifugal/centrifugo.git $GOPATH/src/centrifugo
+WORKDIR $GOPATH/src/centrifugo
+
+COPY main/centrifugo main.go
```

Comment:
Can we do something like this instead:
```
RUN inject-profiler-code.sh main.go
```

Reasons I think this would be better:
* when consul, centrifugo, etc make changes to their main.go files this still works
* from copyright perspective this seems like a cleaner solution, because this way we don't need to copy other people's code into our repo.


I don't want to copy